app/src/components/blocks/_delete/removeRecordStage/RemoveRecordStage.tsx

```diff
@@ -0,0 +1,179 @@
+import { buildSearchResult } from '../../../../helpers/test/testBuilders';
+
+export type Props = {
+    setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
+    recordType: string;
+};
+
+function RemoveRecordStage({ setStage, recordType }: Props) {
+    useTitle({ pageTitle: 'Remove record' });
+    const patientDetails = usePatient();
+    const [submissionState, setSubmissionState] = useState(SUBMISSION_STATE.PENDING);
+
+    const baseUrl = useBaseAPIUrl();
+    const baseHeaders = useBaseAPIHeaders();
+    const nhsNumber: string = patientDetails?.nhsNumber ?? '';
+    const navigate = useNavigate();
+
+    const [searchResults, setSearchResults] = useState<Array<SearchResult>>([]);
+
+    const mounted = useRef(false);
+    useEffect(() => {
+        const onSuccess = (result: Array<SearchResult>) => {
+            setSearchResults(result);
+            setSubmissionState(SUBMISSION_STATE.SUCCEEDED);
+        };
+
+        const onPageLoad = async () => {
+            try {
+                const results = await getDocumentSearchResults({
+                    nhsNumber,
+                    baseUrl,
+                    baseHeaders,
+                });
+                onSuccess(results ?? []);
+            } catch (e) {
+                const error = e as AxiosError;
+                if (isMock(error)) {
+                    onSuccess([buildSearchResult(), buildSearchResult()]);
+                } else if (error.response?.status === 403) {
+                    navigate(routes.SESSION_EXPIRED);
+                } else if (error.response?.status && error.response?.status >= 500) {
+                    navigate(routes.SERVER_ERROR + errorToParams(error));
+                } else {
+                    setSubmissionState(SUBMISSION_STATE.FAILED);
+                }
+            }
+        };
+        if (!mounted.current) {
+            mounted.current = true;
+            void onPageLoad();
+        }
+    }, [
+        patientDetails,
+        searchResults,
+        nhsNumber,
+        setSearchResults,
+        setSubmissionState,
+        navigate,
+        baseUrl,
+        baseHeaders,
+    ]);
+
+    const hasDocuments = !!searchResults.length && !!patientDetails;
+
+    return (
+        <>
+            <BackLink
+                data-testid="back-link"
+                href="#"
+                onClick={(e) => {
+                    e.preventDefault();
+                    setStage(LG_RECORD_STAGE.RECORD);
+                }}
+            >
+                Go back
+            </BackLink>
+            <h1>Remove this {recordType}</h1>
+            <WarningCallout>
+                <WarningCallout.Label>Before removing</WarningCallout.Label>
+                <p>
+                    Only permanently remove this patient record if you have a valid reason to. For
+                    example, you confirmed these files have reached the end of their retention
+                    period.
```

Comment:
In Figma design, the text "retention period" is linked to an external site:
https://transform.england.nhs.uk/information-governance/guidance/records-management-code/records-management-code-of-practice/#appendix-ii-retention-schedule

Reply:
Added